server/lib/src/valueset/address.rs

```diff
@@ -286,6 +292,44 @@ impl ValueSetEmailAddress {
     }
 }
 
+impl ValueSetScimPut for ValueSetEmailAddress {
+    fn from_scim_json_put(value: JsonValue) -> Result<ValueSetResolveStatus, OperationError> {
```

Comment:
What's this intended to do? (Docs plz) 😀 Because an empty put is a delete, but this throws an error